wp-content/themes/eighshift-task/src/blocks/custom/image-text/image-text.js

```diff
@@ -0,0 +1,107 @@
+import { Fragment } from '@wordpress/element';
+import { InspectorControls, BlockControls } from '@wordpress/editor';
+
+import { getActions } from 'EighshiftBlocksGetActions';
+import manifest from './manifest.json';
+
+import { ImageTextEditor } from './components/image-text-editor';
+import { ImageTextOptions } from './components/image-text-options';
+
+import { ImageOptions } from '../../components/image/components/image-options';
+import { HeadingOptions } from '../../components/heading/components/heading-options';
+import { ParagraphOptions } from '../../components/paragraph/components/paragraph-options';
+import { ButtonOptions } from '../../components/button/components/button-options';
+
+import { HeadingToolbar } from '../../components/heading/components/heading-toolbar';
+import { ParagraphToolbar } from '../../components/paragraph/components/paragraph-toolbar';
+
+export const ImageText = (props) => {
+  const {
+    setAttributes,
+    attributes: {
```

Comment:
if you desctructure additional key here as attributes. then you can pass only attributes key in ImageTextOptions